showcase/src/app/app.module.ts

```diff
@@ -93,10 +96,10 @@ export function starkAppConfigFactory(): StarkApplicationConfig {
 
 	const applicationConfig: StarkApplicationConfig = Deserialize(config, StarkApplicationConfigImpl);
 
-	applicationConfig.rootStateUrl = "home";
+	applicationConfig.rootStateUrl = "/";
 	applicationConfig.rootStateName = "";
 	applicationConfig.homeStateName = "home";
-	applicationConfig.errorStateName = "";
+	applicationConfig.errorStateName = starkLoginStateName;
```

Comment:
Maybe you can use the `no-content` component we already have in the `showcase/src/app/no-content/no-content.component.ts` so at least it is clear it is an invalid state. 

We can fix this later on when we implement our own error state page....